### PR TITLE
Nav menu

### PR DESCRIPTION
**Pull request by Owusu-Desmond**
# Description

In this pull request I add functionality to my mobile menu icon

Fixes # (issue)

## Type of change
- [ ] Create mobile menu pop up
- [ ] Style style menu pop up
- [ ] Write script to make the mobile menu pop up hide and show when a user click on the menu icon.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Owusu-Desmond and @amejid, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used


# Required Changes ♻️

- [ ] Kindly ensure your pull request summary reflects a team effort and not an individual effort as it is currently.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: css/style.css [hunk 1]
```diff
@@ -508,19 +521,34 @@ summary:hover .summary-chevron-up svg {
 
 @media screen and (min-width: 768px) {
   .background {
-    background-image: url('../images/Header-bg-desktop.svg');
+    background-image: url("../images/Header-bg-desktop.svg");
```
Comment (BenMukebo) [inline css/style.css:524]: - [ ] Great job so far! 👏🏻  however according to the project requirement, on the desktop size the closed button(X), and the menu page must not be visible. Kindly correct this. You can use your @media screen and (min-width: 768px) to display none of your navigation menu page when you're on the desktop size. 😄

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi Team,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- Both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository have been set up correctly. ✔️  
- Good commit history ✔️ 
-  When the user clicks the hamburger button, the mobile menu appears.✔️ 
- When the user clicks the close (X) button, the mobile menu disappears.✔️ 
- When the user clicks any of the mobile menu options, a correct part is displayed and the mobile menu disappears 👏🏻 
- You have followed  JavaScript best practices. ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (Owusu-Desmond) [reply css/style.css:524]: Capture

Hello @BenMukebo  

Can you further explain the changes over there. As you can see the screenshot there is no visible closed button(X), and the menu page on the desktop version.

Comment (BenMukebo) [reply css/style.css:524]: Hello @Owusu-Desmond, 

What I mean, Let's try to open the mobile menu page when you're on the mobile size then try to increase your page until > 786px you are going to see that the navigation menu page will still be visible and showing up instead it close automatically

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @Owusu-Desmond, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**